src/channels/components/AssignmentList/AssignmentList.tsx

```diff
@@ -17,21 +19,46 @@ const messages = defineMessages({
 });
 
 const AssignmentList: React.FC<AssignmentListProps> = props => {
-  const { items, itemsName, totalCount = 0, removeItem } = props;
+  const { items, itemsName, totalCount = 0, removeItem, reorderItem } = props;
 
   const intl = useIntl();
   const classes = useStyles();
   const expanderClasses = useExpanderStyles();
 
+  const handleSortStart = () => {
+    document.body.classList.add(classes.grabbing);
```

Comment:
`react-sortable-hoc` really doesn't add `cursor: pointer` to body?

Reply:
Yes, it is missing that